### DISCUSSION TITLE
Publish Custom Domain: Stuck on 'Your domain status is pending' step

### DISCUSSION THREAD

**Discussion by neziry** [GitbookIO/community#399] [0. General bugs and questions]
Hi,

When i try to publish my docs on custom domain, stuck on this screen. In additional, our domain at Cloudflare and we setted proxying is disabled. Do you have any guidance? Thanks in advance. 

<img width="875" alt="image" src="https://github.com/GitbookIO/community/assets/6665966/04974f26-7b7d-47b1-b56f-45a576495205">

**Comment by aikoii** [comment]
👋 @neziry! Just checked on your setup for integration.yemeksepeti.com and see the CNAME isn't set up correctly yet. Currently, it points to maya.ns.cloudflare.com. dns.cloudflare.com, [which isn't right](https://www.whatsmydns.net/#CNAME/integration.yemeksepeti.com).

To get things working, you'll need to update your DNS settings. Our [guide](https://docs.gitbook.com/publishing/custom-domain/configure-dns#configure-a-cname-record) has all the details, but here's a quick rundown:

Type: CNAME
Name: integration.yemeksepeti.com (or just 'integration' depending on your DNS provider)
Value: Should point to [your-specific]-hosting.gitbook.io (you'll find this unique address in your GitBook settings)

After updating, give it some time for the changes to propagate.
​
